app/components/main-content/PrelaunchSignUpForm.tsx

```diff
@@ -79,14 +63,8 @@ const PrelaunchSignUpForm: React.FC<PrelaunchSignUpFormProps> = ({ setFlowState
 
           <h2>Get exclusive early access to try our product</h2>
 
-          <section aria-label="Sign Up with Google or Github">
-            <h3>sign up with google or github</h3>
-            <Button onClick={signUpWithGoogle}>Continue with Google</Button><br/>
-            <Button onClick={signUpWithGitHub}>Continue with Github</Button>
-          </section>
-
           <section aria-label="Sign Up with Email">
-            <h3>or sign up with email</h3>
+            <h3>Sign up with email</h3>
```

Comment:
Since Darren is working on styling it may be better not to make any changes to the UI unless it is related to this PR.